tests/goss/sanitise.yaml

```diff
@@ -25,7 +25,7 @@ file:
   /app/web/web.config:
     exists: false
   /app/web/robots.txt:
-    exists: false
+    exists: true
```

Comment:
the robots.txt file won't exist in the test environment - there is no "production" flag - we should remove this check for the time being.